content/concepts/interoperability/cross-chain-bridges.md

```diff
@@ -0,0 +1,110 @@
+
+* *Cross-chain claim ID*: A special identifier used for cross-chain transfers. A cross-chain claim ID represents *one* cross-chain transfer of value.
+
+## How Do Cross-Chain Transactions Work?
+
+### Prerequisites
+
+Ensure that the following are set up and running before initiating cross-chain transactions. 
+
+* The locking and issuing chains are both up and running.
+* The witness server(s) are up and running.
+* Set up a bridge between the two chains, including a _door account_ on each chain. On one chain, the asset is locked and unlocked, hence the name "locking chain", and on the other chain, assets are minted and burned, or issued and reclaimed, hence the name "issuing chain".
+* Enable the `XChainBridge` amendment in the `rippled.cfg` configuration file on the locking chain and the issuing chain. 
+
+### How Do Cross-Chain Transactions Work ?
+ 
+At a high-level, cross-chain transactions involve the following steps: 
+
+1. Claim a cross-chain claim ID on the issuing chain.
+2. Submit a commit transaction on the locking chain, attaching the claimed cross-chain claim ID and include a reward for the witness servers. This locks the asset on the locking chain.
+3. Obtain the attestations from the witness servers that the transaction occurred on the issuing chain.
+4. When there are enough signatures to reach quorum, the XRP is automatically released on the issuing chain to the destination account. In some cases, for example a trustline is not set up properly, or a deposit auth is set up, you need to submit a transaction claim for the transferred value on the issuing chain, attaching the attestation as proof that the value was indeed transferred.
+6. The rewards are then distributed to the witness servers' accounts on the issuing chain.
+
+Consider an example where Alice wants to send XRP from her account on the XRP Ledger Mainnet to her account sAlice on a sidechain. In this example. the XRP Ledger Mainnet is the locking chain and the sidechain is the issuing chain. 
+
+For the sake of this example, let's assume that the following prerequisites have been met.
+
+* The sidechain has been configured with validators and is up and running.
+* A bridge that transfers XRP between the locking chain and the issuing chain has been set up with `XChainCreateBridge`.
+* Witness servers are up and running.
+* Alice has already run a `XChainCreateAccountCommit` transaction and has an account on the sidechain.
+
+<!-- Add image of just the bridge created-->
+
+![Cross-chain Transactions](img/xrpl-bridging-solution.png "Cross-chain transactions")
+
+1. sAlice first checks out a claim ID with `XChainCreateClaimID` on the sidechain, specifying the above bridge. She retrieves the claim ID from the transaction metadata or the `xchaincreateclaimid` RPC call.
```

Comment:
[Avoid the term "RPC call"](https://github.com/XRPLF/xrpl-dev-portal/blob/master/CONTRIBUTING.md#terminology). Use "API method" instead. Also, looks like there's a typo ("sAlice").

```suggestion
1. Alice first checks out a claim ID with `XChainCreateClaimID` on the sidechain, specifying the above bridge. She retrieves the claim ID from the transaction metadata or the `xchaincreateclaimid` API method.
```

When the `xchaincreateclaimid` API method has docs, this would also be a good place to link them.